kubernetes/thelounge.yml.erb

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: service
+spec:
+  selector:
+    app: thelounge
+  ports:
+    - port: 80
+      targetPort: 9000
+--
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment
+  labels:
+    app: thelounge
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: thelounge
+  template:
+    metadata:
+      labels:
+        app: thelounge
+    spec:
+      containers:
+        - name: thelounge
+          image: "tonylianlong/thelounge:1.0.0"
+          resources:
+            limits:
+              memory: 2048Mi
+              cpu: 750m
+          ports:
+            - containerPort: 80
+      dnsPolicy: ClusterFirst
+      dnsConfig:
+        searches:
+          - ocf.berkeley.edu
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: virtual-host-ingress
+spec:
+  rules:
+    - host: irc.dev-kubernetes.ocf.berkeley.edu
```

Comment:
I think you can go ahead and add irc.ocf.berkeley.edu here as well, it won't do anything until we make the change in puppet

Reply:
It seems like there is no easy way to just add a host and I don't think I need to copy the whole thing now. https://github.com/kubernetes/kubernetes/issues/43633

Reply:
I think it's fine to keep it as the dev URL until we actually want to transfer it to the final domain